src/index.html

```diff
@@ -7,6 +7,130 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar-start-day--mon calendar-month-length--31">
+      <div class="calendar__day">
+        <div class="calendar__1"></div>
+      </div>
+
+      <div class="calendar__day">
+        <div class="calendar__2"></div>
+      </div>
+
+      <div class="calendar__day">
+        <div class="calendar__3"></div>
+      </div>
```

Comment:
```suggestion
    <div class="calendar calendar-start-day--mon calendar-month-length--31">
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
There is no need to nest more divs inside